README.md

```diff
@@ -3,7 +3,7 @@
 ### This game should allow you to play Tic Tac Toe from the command line.
 
 ### To Play the Game:
-Type `python src/game.py` into your command line prompt while in this repo's directory
+Type `python tic_tac_toe.py` into your command line prompt while in this repo's directory
```

Comment:
This README change should have been its own commit but it got mixed in here.  In the future I will separate the concerns.